hw03_karimh.py

```diff
@@ -8,20 +8,111 @@
 
 import turtle
 
-def function_1():
-    "include docstring"
+#Assigning name to the turtle
+Heer = turtle.Turtle()
 
+def tyre_1(rad, th):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
+    "Makes the left tyre of the car"
 
-def function_2():
-    "docstring"
+    #Gives black color to the tyre
+    Heer.fillcolor("black")
+    Heer.begin_fill()
 
-def function_3():
-    "docstring"
+    #Attributes of the tyre
+    Heer.pensize(th)
+    Heer.circle(rad)
+    Heer.end_fill()
+def tyre_2(rad, th):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
 
-def fucntion_4():
-    "docstring"
+    Heer.fillcolor("black")
+    Heer.begin_fill()
+
+    Heer.pensize(th)
+    Heer.circle(rad)
+    Heer.end_fill()
+def lower_body(l, w):
```

Comment:
Check PEP8 warning; functions should have two blank lines before them.

Reply:
Resolved all PEP8 warnings, added a space between # and the first work of the comment and added two blank lines before functions.